Mac/Extras.install.py

```diff
@@ -9,10 +9,8 @@
 debug = 0
 
 def isclean(name):
-    if name == 'CVS': return 0
-    if name == '.cvsignore': return 0
-    if name == '.DS_store': return 0
```

Comment:
I am not sure that `.DS_store` is even valid. 
All files created by macos are named `.DS_Store`.

But, macos filesystem is not sensible to registry, so it might be better to use `name.lower() == '.ds_store` to cover all possible cases.